ccd-definition/CaseEvent/PREPARE_FOR_HEARING.json

```diff
@@ -145,5 +145,47 @@
     "EndButtonLabel": "Save and continue",
     "ShowSummary": "Y",
     "ShowEventNotes": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
I'm guessing this duplication was to deal with the issue of not being able to hide the complex type properly

Reply:
yep 👍